packages/frontend/src/components/fullscreen-modal/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { ReactNode } from "react";
+import { Navbar, NavbarProps } from "../ui/navbar";
+import { animated } from "@react-spring/web";
+import { cva } from "class-variance-authority";
+
+const rootStyles = cva(
+    ["fixed", "top-0", "left-0", "h-screen", "w-screen", "overflow-y-auto"],
+    {
+        variants: {
+            bgColor: {
+                orange: ["bg-orange"],
+                green: ["bg-green"],
+            },
+        },
+    }
+);
+
+interface FullscreenModalProps {
+    bgColor?: NavbarProps["bgColor"];
+    style: object;
```

Comment:
Maybe we can improve the typing here? The `object` could be replaced with `SpringValues` I think.

Reply:
Absolutely, addressed in 78d4a2ad972972ec0f1b83234b460c01cc4b4497